defactor-com/ui-kit
# feat(CardComponent): add hover effect

## Add hover effect

### What does this PR do?

- Resolve #142 

### Steps to test

1. Run project
2. Check changes

Before:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/cece861f-075e-4c4a-9687-6018d9c92e31)

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/5b96a727-fa06-482a-8442-952fc81c4f75)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/87fbf406-c46f-4242-b130-2f819bb73da4)


#### CheckList
